themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties

```diff
@@ -99,10 +99,24 @@ revoke=R\u00e9voquer un droit
 
 configureAuthenticators=Authentifications configur\u00e9es.
 mobile=T\u00e9l\u00e9phone mobile
-totpStep1=Installez <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> ou bien Google Authenticator sur votre mobile. Ces deux applications sont disponibles sur <a href="https://play.google.com">Google Play</a> et Apple App Store.
+totpStep1=Installez une des applications suivantes sur votre mobile
```

Comment:
Now we miss the links, it needs to be escaped like here https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources-community/theme/base/account/messages/messages_es.properties#L98